src/mlpack/tests/main_tests/preprocess_encoding_string_test.cpp

```diff
@@ -0,0 +1,411 @@
+/**
+ * @file preprocess_encoding_string_test.cpp
```

Comment:
Why is this even in the diff?

Reply:
Oh, wow, I kind of dropped the ball on this one.  It looks like I built my branch on top of the string encoding PR.  Let me redo this PR...